file/lib/file_source_endpoint.ex

```diff
@@ -29,21 +32,40 @@ defmodule Membrane.RTC.Engine.Support.FileSourceEndpoint do
               ],
               ssrc: [
                 spec: RTP.ssrc_t(),
+                default: 1,
                 description: "SSRC of RTP packets"
               ],
               payload_type: [
                 spec: RTP.payload_type_t(),
+                defalt: 127,
                 description: "Payload type of RTP packets"
+              ],
+              toilet_capacity: [
+                spec: integer(),
+                default: 1_000,
+                description:
+                  "Size of toilet between payloader and realtimer and between realtimer and track_sender"
```

Comment:
What do you think about adding information that size is measured in RTP packets?